demisto_sdk/commands/pre_commit/.pre-commit-config_template.yaml

```diff
@@ -0,0 +1,80 @@
+exclude: .devcontainer/|.vscode|Pipfile.lock|.gitlab/ci/|.circleci/
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-json
+      - id: check-yaml
+      - id: check-ast
+      - id: check-merge-conflict
+      - id: debug-statements
+        language_version: python3
+      - id: name-tests-test
+        files: .+_test.py$
+
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.251"
```

Comment:
```suggestion
    rev: "v0.0.253"
```

Reply:
Can it be automated? e.g. poetry and/or dependabot?